examples/06_second_level_models_non_parametric_tests/plot_thresholding.py

```diff
@@ -0,0 +1,80 @@
+"""
```

Comment:
I have the feeling that that one is not adding anything, right ?
Could be skipped then.

Reply:
Deleted